adam-core/src/main/scala/org/bdgenomics/adam/models/Interval.scala

```diff
@@ -41,4 +42,19 @@ trait Interval {
    */
   def width: Long = end - start
 
+  /**
+   * Need to implement getStart function from Locatable. (1-based start position. closed)
```

Comment:
I guess I don't see where it is used, so I'm not sure what suggestion to make.  Will have to dig a bit.